docs/GLOSSARY.md

```diff
@@ -0,0 +1,97 @@
+
+  <dt>RxNorm, RxCUI</dt>
+  <dd><em>RxNorm</em> is a US-specific standardized naming system for
+  medications. RxNorm also defines IDs for medications, known as
+  <em>RxCUI</em>s.</dd>
+  <dd><a href="https://www.merriam-webster.com/dictionary/Rx">Trivia</a>: Rx
+  stems from the latin word <em>recipe</em> meaning "take". The first doctor to
+  use "Rx" used it as a verb with the same meaning in "Rx two aspirin." on a
+  prescription being equivalent to today's "Take two aspirin.". RxCUI stands for
+  "Rx concept unique identifier".</dd>
+
+  <dt>Indication</dt>
+  <dd>Defines the reason a medication is used, e.g. a headache for
+  ibuprofen.</dd>
+</dl>
+
+## Biomedical terms
+
+<dl>
+  <dt>DNA</dt>
+  <dd><em>DNA</em> is the sequence of base pairs that makes up a human's genetic
+  code.</dd>
+
+  <dt>Gene</dt>
+  <dd>A specific section of the DNA defining how one enzyme is built. The gene
+  <em>CYP2D6</em> for example encodes the enzyme <em>CYP2D6</em>. A gene's name,
+  for example <em>CYP2D6</em>, is often referred to as a
+  <code>GeneSymbol</code>.</dd>
+
+  <dt>Genotype</dt>
+  <dd>What exactly a specific gene looks like, i.e. which exact base pairs it's
+  made up of, may differ for each individual. This specific sequence of base
+  pairs or <em>variant</em> is referred to as the gene's <em>genotype</em>.</dd>
+
+  <dt>Phenotype</dt>
+  <dd>In practice, the different genotypes can have different effects on the
+  encoded enzyme. The <em>phenotype</em> describes this effect for a given gene.
+  One possible phenotype for CYP2D6 is "Normal metabolizer".</dd>
+
+  <dt>Allele</dt>
+  <dd>In theory, a person has two copies of each gene: one from each parent.
+  These copies are called <em>alleles</em>. In reality, the exact number of
+  inherited alleles by each parent may be unequal to one. The major way we
+  express an allele's genotype is by using the <em>star allele</em> format, e.g.
+  <em>CYP2D6*1</em> for CYP2D6's star one variant, which is often used as the
+  baseline or "normal" variant of a gene.</dd>
+
+  <dt>Haplotype</dt>
+  <dd>Describes the allele(s) inherited by a single parent. If an individual has
+  multiple alleles from the same parent, the tar allele format joins them with a
+  hyphen, e.g. <em>CYP2D6*1-*2</em> for a haplotype with one CYP2D6 star one and
+  one CYP2D6 star two allele.</dd>
+
+  <dt>Diplotype</dt>
+  <dd>Describes the allele(s) inherited by both parents, i.e.  the two
+  haplotypes. The star allele format defines diplotypes by joining the
+  respective haplotypes with a slash, i.e. <em>CYP2D6*1/*2</em> for an
+  individual that has one CYP2D6 star one variant from one parent and one star
+  two variant from the other.</dd>
+</dl>
+
+## Pharmacogenomic terms
+
+<dl>
+  <dt>Implication</dt>
+  <dd>For a given medication-phenotype pair, an implication describes what
+  effects the phenotype has on use of the medication. An example for an
+  implication is "Increased risk of side effects".</dd>
+
```

Comment:
Add annotation (a medication with all its guidelines)

Reply:
As far as I'm aware, this currently isn't implemented so I also wouldn't add it to the glossary until it is. If you are currently implementing this, I suggest we merge this PR now and you add *annotation* to the glossary along with its implementation when it's ready.